coucharchive

```diff
@@ -661,6 +697,11 @@ def main():
         '--ideal-duration', dest='ideal_duration', action='store',
         help='optimize concurrent replications and server load to finish in N '
         'seconds', default='fastest', type=check_ideal_duration)
+    sub['create'].add_argument(
+        '--no-run-queue-control', dest='no_run_queue_control',
+        action='store_true', default=False,
+        help='disable throttling of requests if CouchDB `run_queue` is too '
+             'high')
```

Comment:
Since #22, there is no real "throttling of requests", so I propose something like "don't read CouchDB run_queue length to adapt the number of concurrent replications", what do you think?

Reply:
Good catch! Your proposal looks ok to me, I will use it, thanks!